x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -2,6 +2,11 @@
 [[fips-140-compliance]]
 === FIPS 140-2
 
+NOTE: If you consider configuring {es} for FIPS 140-2 compliance, we stronly
+suggest that you first upgrade your installation to 7.11 or later.
+See https://www.elastic.co/guide/en/elasticsearch/reference/current/release-notes-7.11.0.html[7.11 release notes]
```

Comment:
I don't think we should link to current, since that will move and eventually not be 7.11 anymore:
```suggestion
See https://www.elastic.co/guide/en/elasticsearch/reference/7.11/release-notes-7.11.0.html[7.11 release notes]
```